composer.json

```diff
@@ -42,7 +42,8 @@
             "src/Core/functions.php",
             "src/Collection/functions.php",
             "src/I18n/functions.php",
-            "src/Utility/bootstrap.php"
+            "src/Utility/bootstrap.php",
+            "src/TestSuite/phpunit-forward-compat.php"
```

Comment:
It would work for cakephp test suite but not for an existing application bootstrap I think